datacite/schemas/datacite-v4.1.json

```diff
@@ -70,7 +70,18 @@
       "type": "array",
       "items": {
         "creatorName": {
-          "type": "string"
+          "type": "object",
+          "properties": {
+            "creatorName": {
+              "type": "string"
+            },
+            "nameType": {
```

Comment:
I think ``nameType`` should be moved out like ``familyName``/``givenNames`` so that we don't change data type of ``creatorName`` on a minor version.

Reply:
:+1: 
Changed to a sibling instead of subfield with a proper serialization.